cli/exchange/business.go

```diff
@@ -94,9 +94,14 @@ func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath
 	//add/overwrite business policy file
 	httpCode := cliutils.ExchangePutPost("Exchange", http.MethodPost, exchUrl, "orgs/"+polOrg+"/business/policies"+cliutils.AddSlash(policy), cliutils.OrgAndCreds(org, credToUse), []int{201, 403}, policyFile, nil)
 	if httpCode == 403 {
-		cliutils.ExchangePutPost("Exchange", http.MethodPut, exchUrl, "orgs/"+polOrg+"/business/policies"+cliutils.AddSlash(policy), cliutils.OrgAndCreds(org, credToUse), []int{201, 404}, policyFile, nil)
-		msgPrinter.Printf("Deployment policy: %v/%v updated in the Horizon Exchange", polOrg, policy)
-		msgPrinter.Println()
+		//try to update the existing policy
+		httpCode = cliutils.ExchangePutPost("Exchange", http.MethodPut, exchUrl, "orgs/"+polOrg+"/business/policies"+cliutils.AddSlash(policy), cliutils.OrgAndCreds(org, credToUse), []int{201, 404}, policyFile, nil)
+		if httpCode == 200 {
+			msgPrinter.Printf("Deployment policy: %v/%v updated in the Horizon Exchange", polOrg, policy)
+			msgPrinter.Println()
+		} else if httpCode == 404 {
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Cannot create deployment policy %v/%v because maxPolicies limit has been reached", polOrg, policy))
```

Comment:
Please display the message returned from the exchange because it contains the limit count.  You can reference this code: https://github.com/open-horizon/anax/blob/master/cli/exchange/node.go#L177-L181

Reply:
Thanks for reference. PR has been updated